data/tutorials/language/3ds_04_hashtbl.md

```diff
@@ -6,42 +6,62 @@ description: >
 category: "Data Structures"
 ---
 
-## Module Hashtbl
+## Introduction
 
-The Hashtbl module implements an efficient, mutable lookup table. To
-create a hash table we could write:
+The OCaml Standard Library's `Hashtble` module implements an associative array
+similar to the standard library's `[Map](https://ocaml.org/docs/maps)` module.
```

Comment:
I think it's way more common to have the backticks only around `Map` like this `[`Map`]`.